src/landusedata/regrid.py

```diff
@@ -27,6 +53,69 @@ def GenerateRegridder(ds_to_regrid, ds_regrid_target, regridder_weights_file, re
 
     return(regridder)
 
+
+def make_se_regridder(weight_file, regrid_method):
+    weights = xr.open_dataset(weight_file)
+    in_shape = weights.src_grid_dims.load().data.tolist()[::-1]
```

Comment:
@maritsandstad I'm hitting an error here when I try and supply a weight file during the two-step process:
```
  File "/home/glemieux/Repos/NGEET/Tools/tools-fates-landusedata/src/landusedata/regrid.py", line 59, in make_se_regridder
    in_shape = weights.src_grid_dims.load().data.tolist()[::-1]
               ^^^^^^^^^^^^^^^^^^^^^
  File "/home/glemieux/local/miniconda3/envs/landusedata-env/lib/python3.13/site-packages/xarray/core/common.py", line 306, in __getattr__
    raise AttributeError(
        f"{type(self).__name__!r} object has no attribute {name!r}"
    )
AttributeError: 'Dataset' object has no attribute 'src_grid_dims'
```
I'm using the output from using the `-w` option from a previous non-two step run.